features/editor_can_set_approval_status_and_comment.feature

```diff
@@ -13,15 +13,16 @@ Feature: Editor can set approval status and comment
         Given the following articles exist
             | header    | subheader            | body                  | category    | status  | comment                              |
             | Top title | A breaking news item | Today at craftacademy | Programming | pending | No comments currently logged         |
```

Comment:
No need to have the subheader and body attributes in here as we don't test for them. We removed these in the other file with this refactoring.